src/docfx/build/link/FileLinkMapBuilder.cs

```diff
@@ -52,6 +54,6 @@ public object Build(PublishModel publishModel)
         var publishFiles = publishModel.Files.Where(item => !item.HasError && item.SourceFile != null).Select(item => item.SourceFile).ToHashSet();
         var links = _links.Value.Where(x => publishFiles.Contains(x.InclusionRoot)).OrderBy(x => x).ToArray();
 
-        return new { links };
+        return new { _hostName, links };
```

Comment:
```suggestion
        return new {  hostName = _hostName, links };
```

Reply:
BTW, why do we need this change? I thought we wouldn't touch `.links.json` for current work.

Reply:
According to @easonyq, this change is to fix a bug in broken link service. It's a small change, so we just cover it in this pr.

Reply:
https://ceapex.visualstudio.com/Engineering/_workitems/edit/720329